src/js/stores/AppsStore.js

```diff
@@ -115,8 +115,15 @@ function getAppHealthWeight(health) {
   }, 0);
 }
 
+function calculateTotalResources(app) {
+  app.totalMem = app.mem * app.instances;
+  app.totalCpus = parseFloat((app.cpus * app.instances).toPrecision(4));
```

Comment:
Ah now I got it. ```(app.cpus * app.instances).toPrecision(4)``` returns a string, but maybe we want to calculate with ```totalCpus```, so that should be parsed to a Number again.